datadog_checks_base/datadog_checks/base/data/agent_requirements.in

```diff
@@ -1,38 +1,37 @@
-adodbapi==2.6.0.7; sys_platform == 'win32'
-aerospike==3.10.0; sys_platform != 'win32'
+adodbapi==2.6.0.7; sys_platform == "win32"
+aerospike==3.10.0; sys_platform != "win32"
 aws-requests-auth==0.4.2
 beautifulsoup4==4.5.1
 binary==1.0.0
-boto==2.46.1
 boto3==1.10.27
+boto==2.46.1
 botocore==1.13.42
 clickhouse-cityhash==1.0.2.3
 clickhouse-driver==0.1.2
-contextlib2==0.6.0; python_version < '3.0'
+contextlib2==0.6.0; python_version < "3.0"
 cryptography==2.8
 cx-oracle==7.2.0
 ddtrace==0.32.2
 dnspython==1.16.0
-enum34==1.1.6; python_version < '3.0'
-flup==1.0.3.dev-20110405; python_version < '3.0'
-flup-py3==1.0.3; python_version > '3.0'
-futures==3.3.0; python_version < '3.0'
-gearman==2.0.2; sys_platform != 'win32' and python_version < '3.0'
+enum34==1.1.6; python_version < "3.0"
+flup-py3==1.0.3; python_version > "3.0"
+flup==1.0.3.dev-20110405; python_version < "3.0"
+futures==3.3.0; python_version < "3.0"
+gearman==2.0.2; sys_platform != "win32" and python_version < "3.0"
 in-toto==0.4.2
-ipaddress==1.0.22; python_version < '3.0'
+ipaddress==1.0.22; python_version < "3.0"
 jaydebeapi==1.2.3
-jpype1==0.7.1 ; python_version <= '2.7'
-# 0.7.1 version breaks in py3 but is the latest py2 compatible version
```

Comment:
can we keep the comment?

Reply:
when someone runs the command this will happen again. since we're pinning version based on the marker, I think it's already explicit that the reason is compatibility. there will definitely be more cases like this. wdyt?